ftpsync/util.py

```diff
@@ -157,7 +157,10 @@ def check_cli_verbose(default=3):
 
 DEFAULT_CREDENTIAL_STORE = "pyftpsync.pw"
 DRY_RUN_PREFIX = "(DRY-RUN) "
-IS_REDIRECTED = os.fstat(0) != os.fstat(1)
+try:
+    IS_REDIRECTED = os.fstat(0) != os.fstat(1)
+except OSError:
+    IS_REDIRECTED = False
```

Comment:
Could you add a short comment here with a reference to the issue?

Reply:
Sure thing